.eslintrc.js

```diff
@@ -133,7 +133,11 @@ module.exports = {
     'unicorn/prefer-module': 'off',
     'unicorn/prevent-abbreviations': 'off',
 
-    'require-eslint-community': ['error']
+    'require-eslint-community': ['error'],
+
+    // FIXME: The version we are currently using is not compatible.
+    // May be removed in #2146. https://github.com/vuejs/eslint-plugin-vue/pull/2146
+    'unicorn/expiring-todo-comments': 'off'
```

Comment:
I don't understand why this is necessary. That rule was enabled for a long time without causing issues.

Reply:
The problem that #2146 fixes is still there, so the lint process now crashes.
We should remove this workaround when we merge #2146.

Reply:
Ah, I see.